### PR TITLE
Docs: add description of single-marker tracking

### PR DESCRIPTION
**Pull request by whoenig**
Addresses issue #484.

### CODE REVIEW

File: docs/configuration.rst [hunk 1]
```diff
@@ -217,6 +231,14 @@ The ``type`` field in the ``crazyflies.yaml`` entries must refer to a type liste
       #. Find the coordinates of the used markers, for example by using ``roslaunch crazyswarm mocap_helper.launch``. (You may need to do ``source ros_ws/devel/setup.bash`` before ``roslaunch``)
       #. Update ``crazyflieTypes.yaml``.
 
+   .. group-tab:: Single Marker
+
+      For single markers, the ``markerConfigurations`` entry simply contains a single ``points`` entry. This point should describe the offset of the marker with respect to the Crazyflie's center of mass. For example, the marker configuration ``"3"`` corresponds to the marker placement shown below:
```
Comment (jpreiss) [inline docs/configuration.rst:236]: Did you mean to add a third `markerConfigurations` entry to the example `crazyflieTypes.yaml`?

File: docs/configuration.rst [hunk 2]
```diff
@@ -82,6 +83,14 @@ Later steps in the documentation will change depending on your selection.
       For example, you might have several standard Crazyflies with arrangement 1,
       and several larger quadcopters with arrangement 2.
 
+   .. group-tab:: Single Marker
+
+      A special case of duplicated marker arrangements is the case where we only use a single marker per robot.
+      As before, the Crazyswarm will use the raw point cloud from the motion capture system and track the CFs frame-by-frame.
+      In this mode, we use optimal task assignment at every frame, which makes this mode more robust to motion capture outliers compared to the duplicate marker arrangements.
+      The main disadvantage is that the yaw angle cannot be observed without moving in the xy-plane.
```
Comment (jpreiss) [inline docs/configuration.rst:91]: Would be nice if we could offer guidance on how long you can hover before yaw drift becomes significant.

**Review by jpreiss — APPROVED** [review]

File: docs/configuration.rst (hunk 1 above)
Comment (whoenig) [reply docs/configuration.rst:236]: No, this is already in there for a while - it was simply not documented (apart from a comment in `crazyflieTypes.yaml`).

File: docs/configuration.rst (hunk 2 above)
Comment (whoenig) [reply docs/configuration.rst:91]: Done!

**Pull request merged**